src/assets/colors.tsx

```diff
@@ -0,0 +1,14 @@
+export default {
+  gray: "#f8f8f8",
+  grayRegular: "#e5e5e5",
+  grayMedium: "#cccccc",
+  graySemibold: "#666666",
+  grayBold: "#333333",
+  purpleLight: "#e2d2ff",
+  purpleRegular: "#6e1fff",
+  redRegular: "#f7625a",
+  yellowRegular: "#fdefd1",
+  yellowMedium: "#fce1a8",
+  yellowBold: "#f1af1b",
+  greenRegular: "#0eda83"
+}
```

Comment:
We tried using elan as a data source instead but it's not in a format that very useful for reaction-force. When we have setup as an exported object, we get autocompletion and compiler help from typescript.